.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - mix local.rebar --force
   - mix deps.get
 script:
+  - MIX_ENV=test mix format --check-formatted
```

Comment:
#### Why not add `credo` to the list, it [can be easily used in a CI](https://github.com/rrrene/credo#exit-status)?
Well... it's not that straightforward. `credo` is available only in `dev` environment, and Travis compiles the project only in the `test` env. We can't simply add `credo` like we added the formatter.

For this config file, Travis will compile in `test`, run the formatter, run the tests, compile in `docs`, run `inch.report` -- ie we are already compiling the project twice.

What should we do about `credo` then?

Reply:
Let's keep it for offline use. Not necessary if we have that as a check in PR checks.